src/content/docs/vulnerability-management/change-vulnerability-status.mdx

```diff
@@ -0,0 +1,203 @@
+* Ignoring a certain kind of vulnerability in bulk so it no longer appears across multiple entities 
+* Removing the Ignored status at a later date if you want to detect those vulnerabilities again
+
+## Ignore a vulnerability [#ignore-vulnerability]
+
+You can manually ignore vulnerabilities on a per entity basis. Once you set the status of an instance to Ignored, we’ll exclude this instance from the summary tiles and vulnerability list by default. Other instances of this vulnerability on other entities will still keep the Affected status.
+
+<CollapserGroup>
+    <Collapser
+        id="ignore-entity"
+        title="Ignore a vulnerability for a specific entity"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All entities** > (select your entity) > **Vulnerability Management** > Click the **Vulnerabilities** tab, then select the vulnerability you want to ignore. If you don't see the vulnerability you want to manage, it might already have the Ignored status. 
+
+    <img
+    title="Vulnerabilities tab scoped to a specific entity"
+    alt="An image showing the scoped vulnerabilities tab of Vulnerability Management"
+    src={vmIgnoreEntityVulnList}
+    />
+
+1. To update the status to Ignored, click the **Change to ignored** button. 
+
+        <img
+            title="Vulnerability instance details pages"
+            alt="An image showing the change to ignored button on vulnerability instance details page"
+            src={vmIgnoreEntityDetails}
+        />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.
+
+    <img
+        title="Vulnerability instance details form"
+        alt="An image showing the ignore form on vulnerability instance details page"
+        src={vmIgnoreEntityForm}
+    />
+
+    </Collapser>
+    <Collapser
+        id="bulk-ignore"
+        title="Ignore a vulnerability across multiple instances"
+    >
+
+1. Go to **[one.newrelic.com](https://one.newrelic.com/) > All capabilities > Vulnerability Management** > Click the **Vulnerabilities** tab 
+1. Click the vulnerability you want to ignore. By default, this vulnerability view is filtered to only show vulnerabilities where at least one entity is marked Affected.
+
+    <img
+        title="Vulnerabilities tab for all entities"
+        alt="An image showing the all entities vulnerabilities tab of Vulnerability Management"
+        src={vmIgnoreBulkVulnList}
+    />
+
+1. Referring to **Area of Impact**, select each entity you want to ignore, then click the **Change to ignored** button.
+    
+    <img
+        title="Vulnerabilities details"
+        alt="An image showing the selection of specific entities"
+        src={vmIgnoreBulkDetails}
+    />
+
+1. Fill out the form to explain the reasoning behind changing this vulnerability to Ignored. Include a time period that the vulnerability should be ignored.
```

Comment:
```suggestion
1. Fill out the form to explain the reasoning behind changing this vulnerability to `Ignored`. Include a time period that the vulnerability should be ignored.
```

Reply:
Standardized this to say "Fill out the form to explain the reasoning behind ignoring th is vulnerability."